morpheus/_lib/include/morpheus/objects/tensor_object.hpp

```diff
@@ -163,6 +163,8 @@ struct ITensor : public ITensorStorage, public ITensorOperations
 
     virtual TensorIndex stride(TensorIndex) const = 0;
 
+    virtual intptr_t stream() const = 0;
```

Comment:
I like this, but it is a slightly different direction than I took in https://github.com/nv-morpheus/Morpheus/pull/730

Where I filled in the `MemDescriptor` struct to hold the stream pointer.

Reply:
This was a quick fix to add the stream in the least amount of changes. We can look at a better implementation in #730